lib/hnvm/config.sh

```diff
@@ -51,6 +51,11 @@ pkg_json="${PWD}/package.json"
 if [[ -f "${pkg_json}" ]]; then
   pkg_json_contents="$(cat "${pkg_json}")"
 
+   echo $pkg_json_contents | jq > /dev/null || {
+    red "An error occurred while parsing package.json"
+    exit 1
+  }
```

Comment:
@josephschmitt @TrevorBurnham I have a feeling this has broken kronos tests running in urbancompass.

More details at https://compass-tech.atlassian.net/browse/BTS-168

Reply:
I've added you both onto the ticket as participants

Reply:
I looked into the issue on the ticket and found a lot of info.

I think we need to change this:

```
   echo $pkg_json_contents | jq '.' > /dev/null || {
    red "An error occurred while parsing package.json"
    exit 1
  }
```

Reply:
https://github.com/UrbanCompass/hnvm/pull/29